app/views/subnets/_fields.html.erb

```diff
@@ -1,20 +1,25 @@
 <%= text_f f, :name %>
+<%= selectable_f f, :type,
+  options_for_select(Subnet.types_with_form_data, @subnet.type),
+  {:label => _("Protocol")},
+  :disabled => !f.object.new_record? %>
 <%= text_f f, :network %>
-<%= text_f f, :mask, :help_inline => _("Netmask for this subnet") %>
+<%= text_f f, :cidr, :label => _("CIDR"), :help_inline => _("CIDR for this subnet") %>
```

Comment:
This is the "prefix length", not the CIDR, the CIDR is `fc00:0:0:55::/64`, but this field refers to the 64 part, thus prefix length.